rtlib/include/rt/Envi.hpp

```diff
@@ -0,0 +1,47 @@
+//
+// Created by Anthony Tamasi on 2020-02-04.
+//
+
+#pragma once
+
+#include <boost/filesystem.hpp>
+#include <envitools/ENVI.hpp>
+#include <opencv2/core.hpp>
+
+#include "rt/Data.hpp"
+
+
+namespace rt {
+    /**
+     * @class ENVI
+     * @brief Represent an ENVI hyperspectral image for the registration pipeline
+     * @ingroup data
+     */
+class ENVI : public Data
+    {
+    public:
+        /** Constructor */
+        explicit ENVI(const boost::filesystem::path& path);
+        //cv::Mat getImage(int idx = 0) { return bands_[idx]; } //Could use this if we read bands into vector. But would also need functions to load into vector, and access vector.
+
+        /** @brief Return a single image from the input data */
+        cv::Mat getImage(int idx = 0);
+
+        /** @brief Return the number of images we have available */
+        int getNumImages() { return envi_.getWavelengths().size(); }
+
+        /** @brief Return whether or not this object has a mesh */
+        bool hasMesh() { return false; }
+
+        /** @brief Return a single mesh from the input data */
+        rt::ITKMesh::Pointer getMesh(int idx = 0) { return nullptr; }
```

Comment:
```suggestion
        ITKMesh::Pointer getMesh(size_t idx = 0);
```

Reply:
Do we not need to return anything for this kind of thing? What would this return?

Reply:
I guess these shouldn't be called from these classes, so they just need to be defined and don't need to return anything. Is that right?

Reply:
Just a heads up, I removed those and got linker errors.